scripts/licenses.sh

```diff
@@ -4,8 +4,6 @@ echo "Extracting third-party licenses..."
 
 license-checker > provisory.txt
 
-license-checker --summary > THIRD_PARTY_LICENSES_SUMMARY.txt
```

Comment:
just to be sure, did you mean to push that?

Reply:
yes, this file is not used anymore